src/app/banner/banner.component.ts

```diff
@@ -19,21 +19,21 @@ export class BannerComponent extends Base implements OnInit {
   versionFromAPI: string;
   versionBuiltWith: string;
 
-  constructor(private metadataService: MetadataService) {
+  constructor(private serviceInfoService: ServiceInfoService) {
     super();
   }
 
   ngOnInit() {
     this.showBanner = environment.staging;
     this.versionBuiltWith = versions.version;
 
-    this.metadataService
-      .getMetadata()
+    this.serviceInfoService
+      .getServiceInfo()
       .pipe(takeUntil(this.ngUnsubscribe))
       .subscribe(
-        (metadata: Metadata) => {
-          if (metadata.hasOwnProperty('version')) {
-            this.versionFromAPI = metadata['version'];
+        (serviceInfo: TRSService) => {
+          if (serviceInfo.hasOwnProperty('version')) {
```

Comment:
I haven't looked closely, but with types you shouldn't need to do this; you should just be able to do `serviceInfo.version`. The existing code was like that, and don't know why it was doing it that way. If you think `version` can be null/undefined (?), you can just do `if (serviceInfo.version) {...`.